hazelcast/src/main/resources/hazelcast-default.xml

```diff
@@ -44,7 +44,9 @@
                 <multicast-port>54327</multicast-port>
             </multicast>
             <tcp-ip enabled="false">
-                <interface>127.0.0.1</interface>
```

Comment:
That is deleted so I expect it is not used anymore and including it will not validate the xml.